qa/tasks/cephfs/test_snap_schedules.py

```diff
@@ -577,6 +585,357 @@ def test_snap_schedule_all_periods(self):
         self.mount_a.run_shell(['rmdir', test_dir])
 
 
+class TestSnapSchedulesSubvolAndGroupArguments(TestSnapSchedulesHelper):
+    def test_snap_schedule_subvol_and_group_arguments_01(self):
+        """
+        Test subvol schedule creation succeeds for default subvolgroup.
+        """
+        self._fs_cmd('subvolume', 'create', 'cephfs', 'sv1')
+        self.fs_snap_schedule_cmd('add', '--subvol', 'sv1', path='.', snap_schedule='1m')
+        self.fs_snap_schedule_cmd('remove', '--subvol', 'sv1', path='.', snap_schedule='1m')
+        self._fs_cmd('subvolume', 'rm', 'cephfs', 'sv1')
+
+    def test_snap_schedule_subvol_and_group_arguments_02(self):
+        """
+        Test subvol schedule creation fails for non-default subvolgroup.
```

Comment:
Do you mean non-existing subvolume group?

Reply:
yeah, I meant non-existing (and non-default) subvolume group.